setup.py

```diff
@@ -14,7 +14,7 @@
         'pytest',
         'pytest-cov',
     ],
-    ':python_version < "3.4"': [
+    'python_version < "3.4"': [
```

Comment:
Hmm: rereading PEP 496, you're right. A little more digging tells me I was misinterpreting the `error in ebb-lint setup command: Invalid environment marker: python_version<"3.4"` exception that tox was throwing. I think it's a version difference: running `tox` under python 2.7 throws the exception, but `python3 -m tox` sets up the virtualenvs and runs the test just fine. 

I'll delete this change line. Any idea about a way to make this not blow up on python 2.7, or is it more like "python3 is what's supported and that's that"?